clang/lib/DPCT/DPCT.cpp

```diff
@@ -614,6 +615,31 @@ int runDPCT(int argc, const char **argv) {
     dpctExit(MigrationSucceeded);
   }
 
+#ifndef _WIN32
+  if (!InterceptBuildCommand.empty()) {
+    SmallString<512> pathToInterceptBuildBinary(DpctInstallPath);
```

Comment:
Algin the code style。
```suggestion
    SmallString<512> PathToInterceptBuildBinary(DpctInstallPath);
```

Reply:
Done.